src/cli/commands/publish.js

```diff
@@ -100,7 +100,7 @@ async function publish(
     body: root,
   });
 
-  if (res != null && res.success) {
+  if (res !== null && res !== false) {
```

Comment:
This requires a comment as you've described in the PR regarding what different registries may return.
Can you check for response code instead?